pkg/trace/flags/flags_windows.go

```diff
@@ -1,31 +0,0 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
-// under the Apache License Version 2.0.
-// This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-2019 Datadog, Inc.
-
-// +build windows
```

Comment:
This was specific for Windows. How do we handle that?

Reply:
This was just moved to `cmd/trace-agent` now.